org.love2d.love2d.metainfo.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.love2d.love2d</id>
+  <name>LÖVE</name>
+  <summary>LÖVE is an awesome 2D game framework for Lua.</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>Zlib</project_license>
+  <content_rating type="oars-1.0"/>
+  <description>
+    <p>LÖVE is an awesome 2D game framework for Lua. </p>
```

Comment:
What's the expected way for a developer to actually run their game in this? A bit more explanation here would ease confusion.